.github/actions/build-json-module-and-redis-with-cargo/action.yml

```diff
@@ -6,7 +6,7 @@ inputs:
   redis-ref:
     description: 'Redis version to build'
     required: true
-    default: '7.2.1'
```

Comment:
Excuse me, do we really need to be that precise to specify `7.4.1` and not just `7.4`?

Reply:
these were the result of a `sed` from 7.2 to 7.4 :shrug: